haproxy.cfg

```diff
@@ -0,0 +1,43 @@
+defaults
```

Comment:
@victorskl I would like to know where does this HAProxy run... does this go to production or it's just to run the mock-prism test endpoints?

Reply:
Nop; not production. Just reverse proxy for mock-prism services for integration tests.